tools/docker/Dockerfile

```diff
@@ -62,6 +62,7 @@ RUN $IDF_PATH/tools/idf_tools.py --non-interactive install required \
 ENV IDF_CCACHE_ENABLE=1
 
 COPY entrypoint.sh /opt/esp/entrypoint.sh
+RUN chmod u+x $IDF_TOOLS_PATH/entrypoint.sh
```

Comment:
Please either change the line above to use $IDF_TOOLS_PATH instead of /opt/esp, or change this line to use /opt/esp instead of $IDF_TOOLS_PATH, to make them consistent.

Reply:
That makes sense and has been adjusted. Thank you @igrr